### PR TITLE
chore: moved today's question into used block

### PR DESCRIPTION
**Pull request by tweigelt-sqsp**
After today's suggestions (when will I have the time to listen to all of these great soundtracks) I took the liberty to move the question into the `used` block.

### CODE REVIEW

**Review by dalyw01 — APPROVED** [review]
Great suggestion, think it'll help minimize noise and repetition.

Think a few more of these have been used too?

File: src/questionGenerator.js [hunk 1]
```diff
@@ -21,6 +20,7 @@ const questions = [
 
 const used = [
   'Window seat or aisle seat?',
+  'Which movie do you think has the best soundtrack?',
```
Comment (mwall01) [inline src/questionGenerator.js:22]: Yep, this one too! 
'What is your go-to breakfast? Is it sweet or savory?',

**Review by mwall01 — APPROVED** [review]

**Comment by tweigelt-sqsp** [comment]
> Think a few more of these have been used too?

I can not recall them feel free to move them (like @mwall01 did --- see https://github.com/mwall01/weekly-wall/pull/3#discussion_r1476007421)
Or add them here and I submit push them into this submission.

File: src/questionGenerator.js (hunk 1 above)
Comment (tweigelt-sqsp) [reply src/questionGenerator.js:22]: Consider it moved

Comment (tweigelt-sqsp) [reply src/questionGenerator.js:22]: Addressed code review change request(s) through: [2923772](https://github.com/mwall01/weekly-wall/pull/3/commits/29237729a25ba6904d22d822c1ea39b2b3aa929c)

Comment (tweigelt-sqsp) [reply src/questionGenerator.js:22]: **Resolving Conversation**

**Review by dalyw01 — APPROVED** [review]
Great work, game-changing feature.

**Pull request merged**